osu.Game/Screens/OnlinePlay/Matchmaking/Screens/Pick/BeatmapSelectionGrid.cs

```diff
@@ -42,6 +45,11 @@ public partial class BeatmapSelectionGrid : CompositeDrawable
 
         private bool allowSelection = true;
 
+        private readonly Sample[] rouletteSamples = new Sample[8];
```

Comment:
I think this needs to be a `Sample?` so it can work without the `osu-resources` branch?

Reply:
`master`'s been updated with the samples so this is a non-issue.